docs/migrations.rst

```diff
@@ -539,6 +539,55 @@ Option    Description
 comment   set a text comment on the table
 ========= ===========
 
+Changing Table Options
+~~~~~~~~~~~~~~~~~~~~~~
+
+To change the options on an existing table, use the ``change()`` method.
+Options ``id`` and ``primary_key`` are used to determine the primary key on the table, same as when creating a new table.
+Also see `Creating a Table`_ for lists of additional supported options for each adapter.
+
+In the following example, a table is initially created with the default primary key, no table comment, and two columns.
+Then another column is added, and the ``change()`` method is used to remove the default primary key,
+set the 'new_id' column to be the new primary key, and add a new comment on the table.
+
+.. code-block:: php
+
+        <?php
+
+        use Phinx\Migration\AbstractMigration;
+
+        class MyNewMigration extends AbstractMigration
+        {
+            /**
+             * Migrate Up.
+             */
+            public function up()
+            {
+                $users = $this->table('users');
+                $users
+                    ->addColumn('username', 'string', ['limit' => 20])
+                    ->addColumn('password', 'string', ['limit' => 40])
+                    ->save();
+
+                $users
+                    ->addColumn('new_id', 'integer', ['null' => false])
+                    ->change([
```

Comment:
I am wondering if we could have specific methods, instead having an array interface. Something like

```
$users->changePrimaryKey()
$users->changeComment()
```

Internally those method could just use `change()`. What do you think?

Reply:
It came across my mind too. On one hand, change() provides a similar interface to Migration::table(), allowing users to change these options in a similar way as when creating the table, and consistency like that is always nice. On the other hand changePrimaryKey and changeComment would allow the user to be more deliberate and precise, make the methods easier to find, and probably leading to fewer errors too. Another option would be to just provide both interfaces, but I haven't thought much about it yet.

I somehow decided on change() initially, until I got feedback from the owners. So it's your call in the end, once you're sure about the approach let me know and I'll rework it in the next few days.

Reply:
I thought about it for a bit and I think having separate methods is better, can you go that way?

Reply:
Can do! I'll get to it during the weekend probably.